src/index.js

```diff
@@ -93,25 +93,15 @@ function injectToBaseOption(seriesNames, currentIndex, options) {
       },
       globe: {
         viewControl: {
-          autoRotate: false
+          autoRotate: true,
+          autoRotateAfterStill: 5,
+          distance: 300
```

Comment:
I don't know how I feel about the autorotate feature. I suppose I'm ok with it but not with it autorotating again after being still. If you want to look at a specific area, this would be annoying.

Reply:
Autorotating tells users that this is a 3D webapp/webpage from their first view.
Most users think the static things in browser is 2D.

Reply:
I will set the time to 30s when autorotating again after being still